development-certs/certs/timur.development.local-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
All development certs have been pulled out into their own top level directory, and are now baked into the common `etna-base` images.  This means we don't have to duplicate these in a few different places, as they are immediately shareable.